src/hooks/useWalletSupportsChain/useWalletSupportsChain.ts

```diff
@@ -32,33 +32,33 @@ import {
   supportsPolygon,
   supportsThorchain,
 } from '@shapeshiftoss/hdwallet-core'
-import { isLedger } from '@shapeshiftoss/hdwallet-ledger'
 import { isMetaMask } from '@shapeshiftoss/hdwallet-shapeshift-multichain'
 import { useMemo } from 'react'
 import { useIsSnapInstalled } from 'hooks/useIsSnapInstalled/useIsSnapInstalled'
 import { selectAccountIdsByChainIdFilter } from 'state/slices/portfolioSlice/selectors'
 import { selectFeatureFlag } from 'state/slices/selectors'
 import { store, useAppSelector } from 'state/store'
 
-type WalletSupportsChainArgs = {
+type WalletDeviceSupportsChainArgs = {
   isSnapInstalled: boolean | null
-  chainAccountIds: AccountId[] // allows dynamic chain-support detection for Ledger
   chainId: ChainId
   wallet: HDWallet | null
 }
 
+type WalletSupportsChainArgs = {
+  chainAccountIds: AccountId[] // allows dynamic chain-support detection for Ledger
+} & WalletDeviceSupportsChainArgs
+
 // use outside react
-export const walletSupportsChain = ({
+export const walletDeviceSupportsChain = ({
```

Comment:
@woodenfurniture what prompted the rename here? I'm not sure about this one - device terminology has clear ties to physical devices, which makes things more confusing

Reply:
Needed to differentiate the naming from `walletSupportsChain` which I opted not to rename to minimise diff (for easier review). Not fussed about what we call it, but "device" doesnt necessarily mean physical